packages/data/lib/repository/preferences_local_repository.dart

```diff
@@ -5,6 +5,9 @@ import 'package:shared_preferences/shared_preferences.dart';
 const _loginKey = 'loggedUserEmail';
 const _passwordKey = 'loggedUserPassword';
 
+const _trendingDateKey = 'trendingDateKey';
+const _comingDateKey = 'comingDateKey';
```

Comment:
store dates in db. preferences are also ok, but let's have more consistent caching flow - store all data in db